### PR TITLE
LPS-122206 SF

### PR DESCRIPTION
**Pull request by CarlosBrichete**

### CODE REVIEW

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path [hunk 1]
```diff
@@ -77,6 +77,31 @@
...
+<tr>
+	<td>FILTER_BY_STATUS_DRAFT_OPTION</td>
+	<td>//*[contains(@class, 'dropdown-item') and contains(text(),'Draft')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FILTER_BY_STATUS_SCHEDULED_OPTION</td>
+	<td>//*[contains(@class, 'dropdown-item') and contains(text(),'Scheduled')]</td>
+	<td></td>
+</tr>
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path:104]: All of these paths look like the same ... so, change in the test you pass what you want and use one unique variable FILER_BY_blablabla (find a good name) and use something like...
`//*[contains(@class, 'dropdown-item') and contains(text(),'${key_filterBySomething}')]`

**Comment by manoelcyreno** [comment]
@CarlosBrichete , take a look at my comment and change it...

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
ci:test:relevant

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by manoelcyreno** [comment]
ci:forward

**Pull request closed without merging**